src/verifier.ts

```diff
@@ -40,13 +40,13 @@ export class Verifier {
   }
 
   /**
-   * Fetches the issuer public key from the issuer.
+   * Get the issuer public key from the issuer.
    * @param sdJwtVC The SD-JWT to verify.
-   * @param issuerPath The issuer path to use.
+   * @param issuerPath The issuer path postfix to .well-known/{issuerPath}, to get the issuer public key. e.g. 'jwt-issuer/user/1234'
    * @throws An error if the issuer public key cannot be fetched.
    * @returns The issuer public key.
    */
-  public async fetchIssuerPublicKeyFromIss(sdJwtVC: JWT, issuerPath: string): Promise<JWK> {
+  public async getIssuerPublicKeyFromWellKnownURI(sdJwtVC: JWT, issuerPath: string): Promise<JWK> {
```

Comment:
@vijayshiyani wait, so you kept this? What if issuer is DID? Haven't we discussed a way to provide a callback for this as well? So end user could configure it the way they want?

Reply:
You can use a verification callback to check a VC, and it needs the issuer's public key as part of the information.

This function makes things easier for verifiers who want to fetch the public key from the issuer's metadata using a well-known URL.